pkg/apis/crd/v1alpha1/types.go

```diff
@@ -51,7 +59,12 @@ type NetworkPolicyRecommendationSpec struct {
 }
 
 type NetworkPolicyRecommendationStatus struct {
```

Comment:
Thanks Yanjun for working this. 
Do you think we can add the start and termination time in Status as well? I think we might need these two information for CLI side?

Reply:
I notice the list command in CLI will display a completion time. Currently this timestamp is retrieved from the column `timeCreated` in the ClickHouse. I have created a `timeCreated` field in the `RecommendedNetworkPolicy` when reading from the ClickHouse. Do you still need another start/end time of the `NetworkPolicyRecommendation` for CLI?

Reply:
I think we might need creationTime as well?
`sparkApplicationTable := [][]string{{"CreationTime", "CompletionTime", "ID", "Status"}}`

Reply:
I see. Let me add them.